taskcluster/ac_taskgraph/transforms/maybe_nightly.py

```diff
@@ -0,0 +1,26 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""
+Apply some defaults and minor modifications to the jobs defined in the maybe-nightly
+kind.
+"""
+
+from taskgraph.transforms.base import TransformSequence
+from taskgraph.util.schema import resolve_keyed_by
+
+
+transforms = TransformSequence()
+
+
+@transforms.add
+def resolve_keys(config, tasks):
+    for task in tasks:
+        for key in ("worker.env.HOOK_PROJECT", "scopes"):
```

Comment:
Note to self: it would be great if taskgraph could automatically `resolve_keyed_by`. Many transforms exist just for this sole purpose.